tests/integration/helpers.py

```diff
@@ -116,17 +115,19 @@ def get_kafka_zk_relation_data(unit_name: str, model_full_name: str) -> Dict[str
     return zk_relation_data
 
 
-def get_provider_data(unit_name: str, model_full_name: str) -> Dict[str, str]:
+def get_provider_data(
```

Comment:
Should this default to `kafka-client` if it is not a valid interface name? I may be wrong on that.

Reply:
Since I added 3 interfaces (`kafka-client-consumer`, `kafka-client-producer`, `kafka-client-admin`) in the testing app, this field need to be specified.